Makefile

```diff
@@ -43,6 +45,9 @@ lint:
 
 precommit: build test fmt lint vet
 
+test-setup:
+	@docker build --no-cache --tag 'test-image' $(TEST_IMAGE_DIR)
```

Comment:
How about we change this into `dunner/test-image`, to avoid possible conflicts?

Reply:
Done